olumidae/QuickCredit
# #166064294 Fixed unresponsive endpoints and wrote unit tests

#What does this PR do? 
Ensures all unresponsive endpoints are working
Ensures unit tests concerning user registration pass

#Description of the task to be completed?
Wrote unit tests and tested with mocha

#How should this be manually tested?
Clone the repo and run.
run "npm test" to ensure unit tests pass

#Any background context you want to provide?
Not applicable

#What are the relevant pivotal tracker stories?
#166064294
